src/warc2zim/content_rewriting/generic.py

```diff
@@ -88,12 +135,36 @@ def rewrite_html(self, head_template: Template, css_insert: str | None):
             self.content_str
         )
 
+    @no_title
     def rewrite_css(self):
-        return ("", CssRewriter(self.url_rewriter).rewrite(self.content))
+        return CssRewriter(self.url_rewriter).rewrite(self.content)
 
-    def rewrite_js(self, opts):
-        rewriter = build_domain_specific_rewriter(self.path, self.url_rewriter)
-        return (
-            "",
-            rewriter.rewrite(self.content_str, opts),
-        )
+    @no_title
+    def rewrite_js(self, opts: dict[str, Any]):
+        ds_rules = get_ds_rules(self.orig_url_str)
+        rewriter = JsRewriter(self.url_rewriter, ds_rules)
+        return rewriter.rewrite(self.content.decode(), opts)
+
+    @no_title
+    def rewrite_jsonp(self):
+        content = self.content.decode()
+        match = JSONP_REGEX.match(content)
+        if not match:
+            return content
+
+        callback = extract_jsonp_callback(self.orig_url_str)
+
+        if not callback:
+            return content
+
+        return callback + match.group(1)
```

Comment:
Let's be consistent and either use `match[1]` here or use `m.group(1)` above

Reply:
Use `m.group(1)` everywhere.